packages/botonic-plugin-contentful/tests/nlp/stemmer.test.ts

```diff
@@ -31,3 +32,25 @@ test.each<any>([
     expect(stemmed).toEqual(expected)
   }
 )
+
+describe('Stemmers', () => {
+  test.each<any>(SUPPORTED_LOCALES)(
+    'TEST numbers in words are kept for locale %s',
+    (locale: Locale) => {
+      // TODO remove when node-nlp PR is merged
+      if (locale == PORTUGUESE) {
```

Comment:
nit: I like to use the strict comparator, `===` instead of `==` when possible

Reply:
I agree. done, thanks